package.json

```diff
@@ -38,7 +39,8 @@
   "lint-staged": {
     "*.{js,jsx,ts,tsx,json,sol}": [
       "prettier --write",
-      "eslint --fix"
+      "eslint --fix",
+      "cspell \"**\""
```

Comment:
this is the commit hook

Reply:
I know that if you include "." in parameters of `prettier --write .` in the context of lint-staged, it actually prettifies every file, per every file committed.

Please be sure that you are using cspell correctly here. My intuition is that you should just write `cspell` instead of `cspell "**"` because I believe that lint-staged passes the changed file path as the last parameter for these.